src/event.rs

```diff
@@ -643,6 +757,31 @@ pub enum KeyCode {
     Null,
     /// Escape key.
     Esc,
+    /// Caps Lock key.
+    ///
+    /// **Note:** this and all following keys can only be read if
+    /// [`KeyboardEnhancementFlags::DISAMBIGUATE_ESCAPE_CODES`] has been enabled with
+    /// [`PushKeyboardEnhancementFlags`].
+    CapsLock,
+    /// Scroll Lock key.
+    ScrollLock,
+    /// Num Lock key.
+    NumLock,
```

Comment:
Is this technically keypad key?

Reply:
I don't think so; they're off the keypad and not marked as keypad keys in the protocol.